comps/chathistory/mongo/docker/Dockerfile

```diff
@@ -6,10 +6,9 @@ FROM python:3.11-slim
 ENV LANG=C.UTF-8
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
-    build-essential \
     libgl1-mesa-glx \
     libjemalloc-dev \
-    vim
+    build-essential
```

Comment:
I understand - vim but why move buidl-essential, and its not even as if alphabetically organized.

Reply:
Good catch!

Reply:
Fixed